src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.MaxConnectionsPerServer.cs

```diff
@@ -47,19 +47,12 @@ public void Set_InvalidValues_Throws(int invalidValue)
         [InlineData(2)]
         [InlineData(int.MaxValue)]
         [InlineData(int.MaxValue - 1)]
+        [OuterLoop("Uses external servers")]
```

Comment:
Where?  It doesn't make any requests at all, does it?

Reply:
That was mistake. This was really experiment to see if the notes about curl are still valid.